pip_build.py

```diff
@@ -189,15 +208,34 @@ def install_whl(whl_fpath):
     os.system(f"pip3 install {whl_fpath} --force-reinstall --no-dependencies")
 
 
+def validate_rc(rc):
+    rc = rc.lower()
+    if rc[:2] != "rc" or not rc[2:].isdigit():
+        raise argparse.ArgumentTypeError(
+            f"--rc value {rc} should be of format rc[0-9]+."
```

Comment:
```
Invalid value for argument `--rc`. Expected a number in the range [0-9]. Received: rc={rc}
```

Reply:
Changed `rc` to `type=int` and used the default validation for `int` thus removing the need for `validate_rc` method.